### DISCUSSION TITLE
Is there a markdown-badges for PyTorch Lighting?

### DISCUSSION THREAD

**Discussion by Keiku** [Lightning-AI/pytorch-lightning#12523] [General]
I can't find the PyTorch Lighting badge in the list below, is it somewhere?

Ileriayo/markdown-badges: Badges for your personal developer branding, profile, and projects. https://github.com/Ileriayo/markdown-badges

**Comment by Borda** [comment]
sure, we have badges coming from the simple icons or you can also use the mention in [EcoCI](https://github.com/PyTorchLightning/ecosystem-ci)
- ![Lightning](https://img.shields.io/badge/-Lightning-792ee5?logo=pytorchlightning&logoColor=white)
```
![Lightning](https://img.shields.io/badge/-Lightning-792ee5?logo=pytorchlightning&logoColor=white)
```
- ![PyTorch Lightning](https://img.shields.io/badge/pytorch-lightning-blue.svg?logo=PyTorch%20Lightning)
```
![PyTorch Lightning](https://img.shields.io/badge/pytorch-lightning-blue.svg?logo=PyTorch%20Lightning)
```
also see: https://github.com/simple-icons/simple-icons/pull/5663

**Reply by Keiku** [reply]
Thank you for your answer. It's very cool.

**Comment by kaparoo** [comment]
How about adding the GitHub link to the badges?

*. [![Lightning](https://img.shields.io/badge/-Lightning-792ee5?logo=pytorchlightning&logoColor=white)](https://github.com/Lightning-AI/lightning)

```markdown
[![Lightning](https://img.shields.io/badge/-Lightning-792ee5?logo=pytorchlightning&logoColor=white)](https://github.com/Lightning-AI/lightning)
```

*. [![PyTorch Lightning](https://img.shields.io/badge/pytorch-lightning-blue.svg?logo=PyTorch%20Lightning)](https://github.com/Lightning-AI/lightning)

```markdown
[![PyTorch Lightning](https://img.shields.io/badge/pytorch-lightning-blue.svg?logo=PyTorch%20Lightning)](https://github.com/Lightning-AI/lightning)
```
